charts/sscs-tribunals-frontend/values.template.yaml

```diff
@@ -18,9 +18,10 @@ nodejs:
     POSTCODE_CHECKER_ENABLED: "true"
     POSTCODE_CHECKER_ALLOWED_RPCS: "birmingham,liverpool,sutton,leeds,newcastle,cardiff"
     SERVICES_IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
-    SERVICES_IDAM_LOGIN_URL: "https://idam-web-public.aat.platform.hmcts.net/login"
+    SERVICES_IDAM_LOGIN_URL: "https://idam-web-public.aat.platform.hmcts.net/register"
     SERVICES_IDAM_SECRET: ${SERVICES_IDAM_SECRET}
     POSTCODE_LOOKUP_TOKEN: ${POSTCODE_LOOKUP_TOKEN}
+    ALLOW_SAVE_RETURN: "true"
```

Comment:
we can remove this from here since we are defining them again in the values.aat.template. We don't have to define it twice.